helpers/DeployHelper.ts

```diff
@@ -137,7 +137,6 @@ export class DeployHelper {
     const stateV2 = await upgrades.upgradeProxy(stateAddress, StateV2Factory, {
       unsafeAllowLinkedLibraries: true,
       unsafeSkipStorageCheck: true,
-      call: { fn: 'setDefaultIdType', args: [NetworkIdTypes.polygonMumbai] },
```

Comment:
Have you checked that this works for upgrade from old to new version of the contracts. `setDefaultIdType` method has the `onlyOwner` modifier but the upgrade method call is done on behalf of the proxy admin contract address, which may cause the tx to revert.
Don't you have any test to check such scenario?

Reply:
It won't work with `onlyOwner`, thant's why it's removed from `upgradeProxy` params